src/concurrency/index.md

```diff
@@ -274,7 +274,7 @@ fn timer() {
 We've moved our `unsafe` code to inside our carefully-planned abstraction,
 and now our application code does not contain any `unsafe` blocks.
 
-This design requires the application pass a `CriticalSection` token in:
+This design requires the application to pass a `CriticalSection` token in:
```

Comment:
```suggestion
This design requires that the application passes a `CriticalSection` token in:
```

Reply:
If we use "that", should not it be "requires that the application pass (not passes)"?

Reply:
Sure, I'm happy with "requires that the application pass a..."

Reply:
Thanks. I changed it to "requires that the application pass a...".